hack/e2e.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -euo pipefail
+
+GOPATH="$(mktemp -d)"
+export GOPATH
+
+ACTUATOR_PKG="github.com/openshift/cluster-api-actuator-pkg"
+
+go get -u -d "${ACTUATOR_PKG}/..."
+
+go test -timeout 60m \
```

Comment:
In a follow up we'll want to consolidate on `actuator-pkg make test-e2e` instead